test/integration_test.go

```diff
@@ -310,10 +298,7 @@ func mustMakeOperators(t *testing.T, cst *coremock.ChainDataMock, logger common.
 			PubIPProvider:           pubIPProvider,
 			OperatorSocketsFilterer: mockOperatorSocketsFilterer,
 		}
-
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, err)
```

Comment:
Is this rechecking a previous error?

Reply:
Not sure why this was needed. I'll remove this one